### ISSUE TITLE
Questions about FlowNetS_BN and FlowNetC_BN

### ISSUE THREAD

**Issue by userb2020** [ClementPinard/FlowNetPytorch#106]
Hello.
Thanks for releasing this amazing code.

I am newly interested in the optical flow area 
and I have some questions about flownet with batch normalization.

I know that in the paper FlowNet, they did not use batch normalization
but in this repository code, there are models with batch normalization.

I would like to know whether batch norm is added by you
and want to know these batch normalization added version is better than the original one in any model.
Also, I want to know why FlowNetS_BN is not good as FlowNetS.

Thank you!

**Comment by ClementPinard** [comment]
Hello, thanks for your interest in this repo.

BatchNorm was added by me. But it turns out it's not very interesting (see [README](https://github.com/ClementPinard/FlowNetPytorch#training-results)). I advice you to keep the original model, which is very simple. FlowNetC is better, but it's also way heavier, so it's up to you.

**Comment by userb2020** [comment]
Thank you for your kind reply!

It was really helpful for me to understand better.

Hope you have a good day!
